app/scripts/models/evaluation/audit.js

```diff
@@ -21,6 +23,40 @@ angular.module('wcagReporter')
         });
     });
 
+    function updateAssertionResult (assertion, data) {
+      var testResult = data.result;
+
+      function composeImportResult (result) {
```

Comment:
- [ ] Should be split up and moved as function to the import; Then calling criterion.addDescription / criterion.updateOutcome, or something similar

Reply:
Later, ...